app/views/admin/conferences/new.html.haml

```diff
@@ -5,7 +5,7 @@
         = f.input :title, hint: "The name of your conference as it shall appear throughout the site. Example: 'OpenSUSE Conference 2013'",
                           input_html: { required: 'required' }
         = f.input :short_title, hint: "A short and unique handle for your conference, using only lower-case letters, numbers and underscores. This will be used to identify your conference in URLs etc. Example: 'froscon2011'",
-                                input_html: { required: 'required' }
+                                input_html: { required: 'required', pattern: '[a-z0-9-]+', title: 'Only numbers, dashes, and/or lower cased letters.' }, :prepend => 'http://myconf.com/'
```

Comment:
Also, the message should be "Only lower-case letters, numbers and underscores" (so that it's consistent with the help message, and it should allow underscores `_` rather than dashes `-`.